src/hevm/hevm-cli/hevm-cli.hs

```diff
@@ -160,10 +160,14 @@ unitTestOptions cmd = do
 
   params <- getParametersFromEnvironmentVariables
 
+  let block = if (512 ==) (testNumber params)
```

Comment:
Not sure if we need to persist 512 as the magic number here. Could just use 0?

Reply:
Ah yes - zero is probably a less magic default - wasn't sure if 512 had some significance :-)